src/mlpack/methods/preprocess/preprocess_split_main.cpp

```diff
@@ -102,11 +116,13 @@ static void mlpackMain()
   // Parse command line options.
   const double testRatio = IO::GetParam<double>("test_ratio");
   const bool shuffleData = IO::GetParam<bool>("no_shuffle");
+  const bool stratifyData = IO::GetParam<bool>("stratify_data");
 
   if (IO::GetParam<int>("seed") == 0)
     mlpack::math::RandomSeed(std::time(NULL));
   else
-    mlpack::math::RandomSeed((size_t) IO::GetParam<int>("seed"));
+    mlpack::math::RandomSeed((size_t)
+     IO::GetParam<int>("seed"));
```

Comment:
```suggestion
    mlpack::math::RandomSeed((size_t) IO::GetParam<int>("seed"));
```
I think so this was correct only, Or I am missing something.

Reply:
Nope your change makes sense. This doesn't cross the 80 character limit.